tooling/Microsoft.VisualStudio.RazorExtension/Microsoft.VisualStudio.RazorExtension.csproj

```diff
@@ -64,9 +64,6 @@
     <Compile Include="RazorInfo\ProjectViewModel.cs" />
     <Compile Include="RazorInfo\RazorInfoToolWindow.cs" />
```

Comment:
I guess you've already done all the work, but might have bee easier to conditionally add files here.